utils.ts

```diff
@@ -289,6 +304,14 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 			overrides.rollup = vitePackageInfo.dependencies.rollup.version
 		}
 
+		// skip if `overrides.esbuild` is `false`
+		if (
+			vitePackageInfo.dependencies.esbuild?.version &&
+			overrides.esbuild !== false
+		) {
+			overrides.esbuild = vitePackageInfo.dependencies.esbuild.version
+		}
```

Comment:
Instead of doing it always, I think it's safer to only do this for esbuild when it's explicitly set to `true`.
```suggestion
		// skip if `overrides.esbuild` is `false`
		if (
			vitePackageInfo.dependencies.esbuild?.version &&
			overrides.esbuild === true
		) {
			overrides.esbuild = vitePackageInfo.dependencies.esbuild.version
		}
```
Because in most cases, the version difference of esbuild doesn't cause a problem.

Reply:
done!